File: split-bill-api/src/main/java/com/shashankbhat/splitbill/dto/BillSaveDto.java
package com.shashankbhat.splitbill.dto;

import com.shashankbhat.splitbill.entity.BillEntity;
import com.shashankbhat.splitbill.entity.BillShareEntity;
import lombok.Data;
import java.util.List;

@Data
public class BillSaveDto {
    BillEntity bill;
    List<BillShareEntity> billShares;
}

File: app/src/main/java/com/shashankbhat/splitbill/database/remote/entity/BillSaveDto.kt
package com.shashankbhat.splitbill.database.remote.entity

import com.shashankbhat.splitbill.database.local.entity.Bill
import com.shashankbhat.splitbill.database.local.entity.BillShare
import kotlinx.serialization.Serializable

@Serializable
data class BillSaveDto (
    var bill: Bill? = null,
    var billShares: List<BillShare>? = null
)
